### ISSUE TITLE
Issues installing

### ISSUE THREAD

**Issue by evanthomjd** [dnstap/golang-dnstap#1]
When trying to install the golang-dnstap utility I receive the following error 

```
package code.google.com/p/goprotobuf/proto: unable to detect version control system for code.google.com/ path
```

Any help with this would be greatly appreciated.

**Comment by edmonds** [comment]
Looks like we need to update to use https://github.com/golang/protobuf.

**Comment by cmikk** [comment]
Fixed in commit bf459cdeab0dd79cafb5a60544bbe44c0564fc1e
